infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -82,6 +92,9 @@
     <logger additivity="false" name="compactor-leader" level="debug">
 	    <appender-ref ref="async_compactor_leader_file"/>
     </logger>
+    <logger name="org.corfudb.infrastructure.NettyServerRouter" level="ERROR" additivity="yes">
```

Comment:
Curious, why to put these errors into SYSLOG rather than corfu logs?

Reply:
It's logged in both SYSLOG and corfu.9000.log `additivity="yes"`